### DISCUSSION TITLE
Error:

### DISCUSSION THREAD

**Discussion by igeogan0402** [AntlerVC/firetable#921] [Support / Q&A] [answered]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Rowy Run version**
If applicable, share the Rowy Run version from your project settings.

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by harinij** [comment, marked as answer]
Hi @igeogan0402 it would be great if you can share a bit more about the issue you are facing or any screenshots. You can also reach out directly on support@rowy.io if you would like us to walk you through any setup process or want to resolve any specific issues over a call. Thanks.
